google/cloud/forseti/services/server.py

```diff
@@ -112,7 +112,7 @@ def get_storage_class(self):
         raise NotImplementedError()
 
 
-class AbstractInventoryConfig(dict):
+class AbstractInventoryConfig(object):
```

Comment:
Will this break other part of the code? if it's not a dictionary you can no longer access using the .get(key) method.

Reply:
Reverted.  Will fix this in another PR.